jinahub/encoders/audio/VGGISHAudioEncoder/vggish_audio_encoder.py

```diff
@@ -2,11 +2,15 @@
 __license__ = "Apache-2.0"
 
 import os
+from pathlib import Path
+
+import requests as requests_
 from typing import Any, Optional, List, Iterable
 import tensorflow as tf
 tf.compat.v1.disable_eager_execution()
 
 from jina import Executor, requests, DocumentArray
+from jina.logging.logger import JinaLogger
 from vggish.vggish_postprocess import *
 from vggish.vggish_slim import *
```

Comment:
can we reorder the imports a bit? 
```python
# os dependencies
import os

# 3rd party dependencies
import requests as requests_
...

# library dependencies
from .helper import helper_func
...
```

Reply:
I have reordered the imports [here](https://github.com/jina-ai/executors/pull/31/commits/d3786b57f0d6474ad54d7060df4f351f0ce62c44). Let me know if this looks okay to you, thank you.